packages/salesforcedx-vscode-apex/src/helpers/languageServerUtils.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  extensionUris,
+  projectPaths
+} from '@salesforce/salesforcedx-utils-vscode';
+import { copyFileSync, existsSync, unlinkSync } from 'fs';
+import { join } from 'path';
+import * as vscode from 'vscode';
+import { VSCODE_APEX_EXTENSION_NAME } from '../constants';
+
+const setupDB = (): void => {
```

Comment:
Just refactored this out of the file of the class that uses it to make it easier to unit test.